scripts/irods/test/session.py

```diff
@@ -60,6 +60,14 @@ def mkgroup_and_add_users(group_name, usernames):
             admin_session.assert_icommand(
                 ['iadmin', 'atg', group_name, username])
 
+def get_data_id(session, collection_name, data_name):
+    rc, out, err = session.assert_icommand(['iquest', "select DATA_ID where COLL_NAME = '{0}' and DATA_NAME = '{1}'".format(collection_name, data_name)], 'STDOUT_SINGLELINE', 'DATA_ID')
+    assert rc == 0, rc
+    assert err == '', err
+    lines = out.split()
+    assert len(lines) == 4, lines # make sure genquery only returned one result
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)